src/devtools/views/Components/OwnersStack.css

```diff
@@ -38,7 +36,48 @@
   outline: none;
 }
 
+.ElementsBar {
+  flex: 1 0 auto;
+}
+
+.ElementsBarSelectedOnly {
+  margin-left: 0.25rem;
+}
+
+.ElementsBarSelectedOnly .Component {
+  visibility: hidden;
+}
+.ElementsBarSelectedOnly .FocusedComponent {
+  float: left;
```

Comment:
Because we are just visually hiding the non-active components we need to move the focused element to the left.